autotest/ogr/ogr_osm.py

```diff
@@ -436,7 +436,7 @@ def ogr_osm_5():
         feat = None
         ds.ReleaseResultSet(sql_lyr)
 
-        if not (test[2] ^ is_none):
+        if not test[2] ^ is_none:
```

Comment:
This change assumes everybody knows operator precedence. I don't... Less risky to have the extra parenthesis